### ISSUE TITLE
Issue with displaying shelters on the SACRAMENTO_USER homepage

### ISSUE THREAD

**Issue by abaranowska1** [benetech/ServiceNet#975]
There is an issue with displaying shelters on the SACRAMENTO_USER homepage and MyShelters listing page. For example, only 20 from 22 records are displayed and I can't load more.
**HOMEPAGE:**
![Peek 2020-05-07 10-01](https://user-images.githubusercontent.com/61964176/81270438-09d32180-904b-11ea-854a-39acec019f38.gif)
**MY SHELTERS:** The spinner icon doesn't disappear.
![Peek 2020-05-07 10-17](https://user-images.githubusercontent.com/61964176/81271166-0b511980-904c-11ea-88dc-d51e59441caa.gif)

What's more, when I go to My Shelters and click the browser's Back button, the progress bar shows the wrong number of records and I see, for example, 40, 60, 80/22 records on the homepage, but in a loop.

![image](https://user-images.githubusercontent.com/61964176/81270802-8d8d0e00-904b-11ea-977e-80f3fa34afaf.png)

![Peek 2020-05-07 09-59](https://user-images.githubusercontent.com/61964176/81270720-73ebc680-904b-11ea-8577-516c4a357d91.gif)

**Steps:** 

1. Log in as SACRAMENTO_USER
2. Scroll the page down 
3. Go to My Shelters 
4. Click Back Button

**Actual result:** 
There is an issue with displaying shelters on the SACRAMENTO_USER homepage and MyShelters listing page. The progress bar shows the wrong number of records.

**Expected result:**
It should work the same as on admin or regular user home page, after scroll - new records should load

**Comment by abaranowska1** [comment]
@kdondziak 
During testing this ticket I saw that you fixed the issue from the description, but another bug appeared after fixes and I guess it could be related to your changes. 
![sacramento](https://user-images.githubusercontent.com/61964176/81695237-a68a2a80-9462-11ea-933b-7d68b921e70a.gif)
During regression testing, we found an issue with sorting #971 , but now sort doesn't work and progress bar display weird numbers. The same problem exists during clicking Reset button 
Plz look at it.
